### ISSUE TITLE
Rolling Ridge logic issue: two-wide gap between ridge base past east and ridge mid past.

### ISSUE THREAD

**Issue by spinerak** [MadagacarDave/AgesTracker#3]
### In short
Hi there, I think I found an issue with the Rolling Ridge logic. I think some two-wide gaps are included in normal/hard logic with just the Feather, while the max jump variable in this tracker is always set to 1 when you only have the Feather (no Pegasus seeds). More precisely: in the west_ridge_present clause, max_jump() >= 2 seems incorrect, as the jumps seems to be in logic with just the Feather.

### I did some more research, so here is some more rambling in case that helps :)
With Feather, Mermaid Suit, Currents and Brother Emblem, one can reach West Rolling Ridge Present (west_ridge_present) in the logic from oracles-randomizer, but not in the logic of the AgesTracker.
This (at least) influences the checks:
Stairs chest - Rolling Ridge Entrance
Surface chairs - Rolling Ridge Entrance
Lower floor chest - Moblin Keep

### Difference in logic.
The difference seems to come from this set of jumps.
![image](https://user-images.githubusercontent.com/12293055/184551951-7a38eba7-4a1e-4a74-82ca-e500d47944eb.png)

This jump seems to be fine in the oracles-randomizer logic, since:
ridge mid past: {or: [(...), [ridge base past east, brother emblem, feather]]}

However in tracker logic, 
west_ridge_present:
return (...) or (lynna() and has("mermaidsuit") and has("currents") and max_jump() >= 2))
max_jump() >= 2 is false in normal 

### Suggestions
My suggestion would be to change "max_jump() >= 2" to "has("feather")"

### Notes
Here are some of my notes for figuring out the full path from start to "ridge west present".
"ridge base past east" is true since [lynna city, or: [feather, ages], mermaid suit] is true.
**"ridge mid past" is true since [ridge base past east, brother emblem, feather]] is true.**
"ridge mid present" is true since [ridge mid past, currents] is true.
"ridge upper present" is true since [ridge mid present] is true.
"ridge west present" is true since [ridge upper present] is true.

I'm having tons of fun with the randomizer, tracker and figuring out how everything works and sometimes diving in, so thanks for all the great resources! :D

**Comment by MadagacarDave** [comment]
yeah, rolling ridge logic is still pretty bad. Would you like to make a Pull Request to fix it? I have another fix for getting to D5 entrance from East Rolling ridge that I also need to merge if you want me to make the change.
